### PR TITLE
Add predictions in the altaz frame

### PR DESCRIPTION
**Pull request by LukasNickel**
Connected Issue: #149

### CODE REVIEW

File: aict_tools/scripts/apply_disp_regressor.py [hunk 1]
```diff
@@ -140,6 +144,8 @@ def main(
             key = config.events_key
             append_column_to_hdf5(data_path, source_x, key, "source_x_prediction")
             append_column_to_hdf5(data_path, source_y, key, "source_y_prediction")
+            append_column_to_hdf5(data_path, source_alt, key, "source_alt_prediction")
```
Comment (maxnoe) [inline aict_tools/scripts/apply_disp_regressor.py:147]: if coordinate transforms are `FACT`, we should fill zenith I think.

Comment (LukasNickel) [reply aict_tools/scripts/apply_disp_regressor.py:147]: Ok

File: aict_tools/scripts/apply_disp_regressor.py [hunk 2]
```diff
@@ -141,10 +140,11 @@ def main(
                 )
 
         elif config.data_format == "simple":
```
Comment (maxnoe) [inline aict_tools/scripts/apply_disp_regressor.py:142]: The simple data format can also use `CTA` coordinate transforms. So you need to check for `coordinate_transforms` inside of `data_format == "simple"`

**Review by maxnoe — COMMENTED** [review]
Let's just call it `{x,y,alt,zd,az}_prediction`. Avoiding source.

**Comment by LukasNickel** [comment]
> Let's just call it `{x,y,alt,zd,az}_prediction`. Avoiding source.

Sure

**Review by maxnoe — APPROVED** [review]

**Pull request merged**